docs/user_guide/nvflare_cli/fl_simulator.rst

```diff
@@ -837,6 +837,11 @@ Specifying threads
 ==================
 The simulator ``-t`` option provides the ability to specify how many threads to run the simulator with.
 
+.. note::
+
+    We use the term threads for simplicity, however technically each client actually runs in a separate process.
+    This difference will not affect the user experience.
```

Comment:
On one hand feels this is OK, the other hand thinking should we update the whole docs of simulator to refer to process?

I guess that's for later, maybe after the Simulator re-design 

@yhwen @chesterxgchen

Reply:
Sure, after the re-design we can update all the docs

Reply:
I think Chester wants to update all the simulator docs?

Reply:
change to use process in the docs, leaving the code ``-t`` flag alone for now